azkaban-common/src/main/java/azkaban/executor/container/ContainerFlowCriteria.java

```diff
@@ -43,35 +43,33 @@ public ContainerFlowCriteria(final Props azkProps, final Logger logger) {
     this.fileLocation = azkProps.getString(Constants.
         ContainerizedDispatchManagerProperties.CONTAINERIZED_FLOW_FILTER_FILE, null);
     this.logger = logger;
-    loadFlowFilter();
+    loadFlowFilter(new HashMap<>());
```

Comment:
Shall we use this https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#emptyMap()? for emptyMap

Reply:
It is immutable. We are intending to populate that map and replace "this.flows" with it.